operator/pkg/lbipam/cell.go

```diff
@@ -118,8 +118,8 @@ func newLBIPAMCell(params lbipamCellParams) *LBIPAM {
 }
 
 const (
-	DefaultLBClassLBIPAM   = "lbipam"
-	DefaulLBClasstNodeIPAM = "nodeipam"
+	DefaultLBClassLBIPAM    = "lbipam"
+	DefaultLBClasstNodeIPAM = "nodeipam"
```

Comment:
nit: Since we're fixing typos, is it supposed to have that "t" in between "Class" and "Node"?

Reply:
Thanks, fixed